dynamic-demo-plugin/src/components/k8sConsumer/k8s-data.ts

```diff
@@ -28,6 +28,7 @@ export const mockDeploymetData = {
             image: 'image-registry.openshift-image-registry.svc:5000/openshift/httpd:latest',
             ports: [
               {
+                protocol: 'tcp',
```

Comment:
This is a required type for DeploymentKind... I found it while mucking around. But we don't want to import the type as it causes the break between monorepo and this test plugin.